canvas_modules/harness/src/client/App.js

```diff
@@ -1980,6 +1981,7 @@ class App extends React.Component {
 				customControls={[CustomToggleControl, CustomTableControl, CustomEmmeansDroplist]}
 				customConditionOps={[CustomOpMax, CustomNonEmptyListLessThan, CustomOpSyntaxCheck]}
 				light={this.state.light}
+				trimSpaces={this.state.trimSpaces}
```

Comment:
Can you follow the pattern we have and put `trimSpaces` as part of the `propertiesConfig`?